exist-versioning-release.md

```diff
@@ -180,11 +180,14 @@ Once development on a new stable version is complete, the following steps will p
 
 #### Publishing the Product Release
 
-1. Login to https://bintray.com/existdb/ and create a new "Version", then upload the files `$EXIST_HOME/installer/eXist-db-setup-3.2.0.jar`, `$EXIST_HOME/dist/eXist-db-3.2.0.dmg`, `$EXIST_HOME/dist/exist-3.1.0.war`, and `$EXIST_HOME/dist/exist-3.1.0.tar.bz2`. Once the files have uploaded, make sure to click "Publish" to publish them to the version. Once published, you need to go to the "Files" section of the version, and click "Actions"->"Show in downloads list" for each file.
+1. Login to https://bintray.com/existdb/ and create a new "Version", then upload the files `$EXIST_HOME/installer/eXist-db-setup-3.1.0.jar`, `$EXIST_HOME/dist/eXist-db-3.1.0.dmg`, `$EXIST_HOME/dist/exist-3.1.0.war`, and `$EXIST_HOME/dist/exist-3.1.0.tar.bz2`. Once the files have uploaded, make sure to click "Publish" to publish them to the version. Once published, you need to go to the "Files" section of the version, and click "Actions"->"Show in downloads list" for each file.
 
 2. Update and publish the latest Maven artifacts as described here: https://github.com/exist-db/mvn-repo
 
-3. Ask [Evolved Binary](http://www.evolvedbinary.com) to build and upload new Docker Images for the latest release.
+3. Use the [docker-existdb](https://www.github.com/eXist-db/docker-exist) to build and upload new Docker Images for the latest release. Run the following commands inside a clone of docker-existdb:
```

Comment:
I think the below is currently more accurate now. However I m working on getting the automated tagging to work, which only leaves the task of merging changes from develop into master (if there are any).

Releases of docker images are automated. If the develop branch of [docker-existdb](https://www.github.com/eXist-db/docker-exist) contains changes that require merging into the master branch, please open a PR. Make sure that the version number in `docker-existdb/hooks/post_push` matches the release number for stable releases. (We are working on automating this).
If you wish to perform a manual release, e.g. for a `release candidate` you need to be registered on docker hub, and a member of the existdb organization there.
To create a release based on a git tag run, e.g.:
1. `docker build --build-arg BRANCH=eXist-3.1.0 --tag existdb/existdb:3.1.0 .`
2. `docker push existdb/existdb:3.1.0`

Reply:
Okay but I think we could merge what I have - as it reflects the current state of play.

Reply:
but it doesn't. Making a manual latest release will not overwrite the automatically released and tagged :release images. The only thing we need to do manually right now is set the version string so that the version string in the docker image tag, actually matches the version of exist running inside it.

Reply:
Okay, any update?

Reply:
@duncdrum so that was merged. What do we do here?

Reply:
@adamretter we finish our epic battle over the tagging conventions